src/ensembl/src/shared/privacy-banner/privacy-banner-service.ts

```diff
@@ -0,0 +1,33 @@
+import storageService, {
+  StorageServiceInterface
+} from 'src/services/storage-service';
+
+export const PrivacyConfig = {
+  name: 'ensembl_privacy_policy',
+  version: '2.0.0',
+  policyUrl: 'https://www.ebi.ac.uk/data-protection/ensembl/privacy-notice',
+  termsUrl: 'https://www.ebi.ac.uk/about/terms-of-use'
+};
+
+export class PrivacyBannerService {
+  private storageService: StorageServiceInterface;
+
+  public constructor(storageService: StorageServiceInterface) {
+    this.storageService = storageService;
+  }
+
+  public getPolicyVersion(): string {
+    return this.storageService.get(PrivacyConfig.name);
+  }
+
+  public setPolicyVersion() {
+    this.storageService.save(PrivacyConfig.name, PrivacyConfig.version);
+  }
+
+  // Returns true if versions doesn't match
```

Comment:
This is an unnecessary comment :-)

Reply:
But if you read the body of this function you see that it returns true if `this.getPolicyVersion()` is the same as `privacyConfig.version` and false otherwise. This comment doesn't add anything to the understanding of the code.

In the test file, we can incorporate the comments into the variable names, which is always better than comments.  For example:

```
      const returnDifferentVersion = () =>faker.lorem.word();
      jest
        .spyOn(privacyBannerService, 'getPolicyVersion')
        .mockImplementation(returnDifferentVersion);

      const result = privacyBannerService.shouldShowBanner();
```

```
      const privacyBannerService = new PrivacyBannerService(mockStorageService);
      const returnSameVersion = () => privacyConfig.version;
      jest
        .spyOn(privacyBannerService, 'getPolicyVersion')
        .mockImplementation(returnSameVersion);

      const result = privacyBannerService.shouldShowBanner();
```